Revit_Core_Engine/Query/DoesIntersect.cs

```diff
@@ -82,30 +82,29 @@ public static bool DoesIntersect(this BoundingBoxXYZ bbox, Element element)
             }
 
             Transform transform = element.Document.LinkTransform();
+            Outline outline = new Outline(bbox.Min, bbox.Max);
```

Comment:
Looks like you are missing the following lines here?

```C#
                if (bbox.Transform.IsTranslation)
                {
                    outline.MinimumPoint += bbox.Transform.Origin;
                    outline.MaximumPoint += bbox.Transform.Origin;
                }
```

I would actually clean up the code a bit:
```C#
Outline outline = new Outline(bbox.Min, bbox.Max);
if (bbox.Transform.IsTranslation)
{
    outline.MinimumPoint += bbox.Transform.Origin;
    outline.MaximumPoint += bbox.Transform.Origin;
}
else
{
    BH.Engine.Base.Compute.RecordWarning("Intersection of the bounding boxes could not be checked. Only translation and identity transformation is currently supported.");
    return false;
}
```

Reply:
I added missing part 👍